2020-05-20 18:06:34,110 ERROR   metis.DAC.view            ['邮箱地址不符合规则']
2020-05-20 18:07:23,986 ERROR   metis.DAC.view            ['两次密码输入不一致']
2020-05-21 09:51:33,886 ERROR   metis.DAC.view            ['两次密码输入不一致']
2020-05-21 14:17:35,659 ERROR   metis.DAC.view            ['邮箱地址不符合规则']
2020-05-25 10:19:11,257 ERROR   metis.DAC.view            ['两次密码输入不一致']
2020-05-26 14:40:37,267 ERROR   metis                     ['账号或密码错误']
2020-05-26 14:42:48,241 ERROR   metis                     ['账号或密码错误']
2020-05-26 14:47:09,323 ERROR   metis                     ['账号或密码错误']
2020-05-26 15:50:27,274 ERROR   metis                     ['邮箱地址已被注册']
2020-05-26 15:53:22,832 ERROR   metis                     ['账户不存在']
2020-05-26 15:55:15,949 ERROR   metis                     ['账户不存在']
2020-05-26 17:16:28,300 ERROR   metis                     Eth地址用完了，暂停注册
2020-05-26 17:17:56,044 ERROR   metis                     Eth地址不足
2020-05-26 17:19:19,595 ERROR   metis                     Eth地址不足
2020-05-27 16:22:43,802 ERROR   metis.DAC.view            Could not persistence user info
2020-05-27 16:25:57,822 ERROR   metis.DAC.view            Could not persistence dac
2020-05-27 16:26:58,869 ERROR   metis.DAC.view            Could not persistence dac
2020-05-27 16:27:41,831 ERROR   metis.DAC.view            Could not persistence dac
2020-05-27 16:30:06,797 ERROR   metis.DAC.view            Could not persistence dac
2020-05-27 16:30:23,700 ERROR   metis.DAC.view            Could not persistence dac
2020-05-27 16:31:45,334 ERROR   metis.DAC.view            ['DAC名称已存在']
2020-05-27 16:37:46,207 ERROR   metis.DAC.view            ['DAC名称已存在']
2020-05-29 11:06:48,056 ERROR   metis.Home.view           ['DAC名称已存在']
2020-06-02 11:55:29,481 ERROR   metis                     Could not persistence work
2020-06-02 11:58:26,465 ERROR   metis                     ['describe uri invalidate', '激励数量不能小于0']
2020-06-02 12:05:01,164 ERROR   metis                     ['激励数量不能小于0']
2020-06-02 12:06:29,024 ERROR   metis                     ['describe uri invalidate']
2020-06-02 12:06:31,077 ERROR   metis                     ['describe uri invalidate']
2020-06-02 12:07:02,679 ERROR   metis                     ['工作描述wiki不正确']
2020-06-02 12:07:27,121 ERROR   metis                     ['描述wiki url格式错误']
2020-06-02 12:07:41,733 ERROR   metis                     ['描述wiki url格式错误']
2020-06-02 12:11:25,644 ERROR   metis                     ['描述wiki url格式错误']
2020-06-02 12:11:33,429 ERROR   metis                     ['描述wiki url格式错误']
2020-06-04 09:39:32,519 ERROR   metis                     taskList add error
2020-06-04 09:40:49,832 ERROR   metis                     ['该工作名已存在']
2020-06-04 09:41:03,836 ERROR   metis                     提交工作到区块链异常
2020-06-04 11:53:14,776 ERROR   metis                     ['该工作无效', '权限不足']
2020-06-04 11:54:01,605 ERROR   metis                     ['该工作无效', '权限不足']
2020-06-04 11:54:42,359 ERROR   metis                     ['权限不足']
2020-06-19 12:17:24,387 ERROR   metis                     ['账户不存在']
2020-06-19 12:17:46,470 ERROR   metis                     ['账户不存在']
2020-06-19 12:17:48,698 ERROR   metis                     ['账户不存在']
2020-06-22 10:33:51,088 ERROR   metis                     Eth地址不足
2020-06-22 10:33:53,512 ERROR   metis                     Eth地址不足
2020-06-22 10:36:45,683 ERROR   metis                     Eth地址不足
2020-06-22 10:36:48,121 ERROR   metis                     Eth地址不足
2020-07-07 12:05:23,673 ERROR   metis                     提交工作到区块链异常
2020-07-07 15:05:35,059 ERROR   metis                     ['余额不足']
2020-07-07 15:06:46,171 ERROR   metis                     ['余额不足']
2020-07-07 15:08:31,096 ERROR   metis                     ['余额不足']
2020-07-07 15:09:43,733 ERROR   metis                     ['余额不足']
2020-07-07 15:09:47,392 ERROR   metis                     ['余额不足']
2020-07-07 15:10:04,498 ERROR   metis                     ['余额不足']
2020-07-07 15:11:07,150 ERROR   metis                     ['余额不足']
2020-07-07 15:15:10,685 ERROR   metis                     ['余额不足']
2020-07-07 15:16:49,192 ERROR   metis                     ['余额不足']
2020-07-07 15:20:51,576 ERROR   metis                     ['余额不足']
2020-07-07 15:24:53,880 ERROR   metis                     ['余额不足']
2020-07-07 15:29:12,122 ERROR   metis                     ['余额不足']
2020-07-07 17:57:23,196 ERROR   metis                     ['余额不足']
2020-07-09 15:52:15,511 ERROR   metis                     eth error
2020-07-09 15:52:44,416 ERROR   metis                     eth error
2020-07-09 16:19:32,372 ERROR   metis                     eth error
2020-07-09 16:57:53,058 ERROR   metis                     Eth Error
2020-07-09 17:01:40,006 ERROR   metis                     Eth Error
2020-07-09 17:01:46,186 ERROR   metis                     Eth Error
2020-07-09 17:02:20,820 ERROR   metis                     Eth Error
2020-07-10 13:16:04,307 ERROR   metis                     ['邮箱地址已被注册']
2020-07-10 14:53:33,438 ERROR   metis                     ['该工作无效']
2020-07-10 14:57:37,074 ERROR   metis                     ['该工作无效']
2020-07-10 19:22:22,154 ERROR   metis                     ['余额不足']
2020-07-12 18:07:06,926 ERROR   metis                     ['余额不足']
2020-07-12 18:13:18,630 ERROR   metis                     Eth Error
2020-07-13 10:20:30,972 ERROR   metis                     ['该工作名已存在']
2020-07-15 17:56:49,431 ERROR   metis                     Eth Error
2020-07-16 22:13:51,194 ERROR   metis                     ['余额不足']
2020-07-17 15:09:52,480 ERROR   metis                     ['余额不足']
2020-07-17 15:11:29,250 ERROR   metis                     ['余额不足']
2020-07-17 15:21:24,669 ERROR   metis                     ['余额不足']
2020-07-17 15:23:00,843 ERROR   metis                     ['余额不足']
2020-07-17 15:24:44,205 ERROR   metis                     ['余额不足']
2020-07-29 13:31:55,297 ERROR   metis                     ['账号或密码错误']
2020-07-29 13:32:31,292 ERROR   metis                     账户余额不足
2020-07-29 13:33:15,804 ERROR   metis                     账户余额不足
2020-07-30 10:48:30,423 ERROR   metis                     账户余额不足
2020-08-14 12:18:17,167 ERROR   metis                     ['账号或密码错误']
2020-08-15 11:48:49,570 ERROR   metis                     (pymysql.err.InternalError) (1054, "Unknown column 'transfer_from' in 'field list'")
[SQL: INSERT INTO metis_transfer_info (transfer_from, transfer_to, transfer_hash, status, amount, ctime, utime) VALUES (%(transfer_from)s, %(transfer_to)s, %(transfer_hash)s, %(status)s, %(amount)s, %(ctime)s, %(utime)s)]
[parameters: {'transfer_from': '0x610Bb1573d1046FCb8A70Bbbd395754cD57C2b60', 'transfer_to': '0x1dF62f291b2E969fB0849d99D9Ce41e2F137006e', 'transfer_hash': '0xb1d8203af3c36fedc4c6116e5688566597527fd45553a3eccec28b58c2f4c885', 'status': 1, 'amount': 100000000000000000000, 'ctime': datetime.datetime(2020, 8, 15, 11, 48, 49, 528279), 'utime': datetime.datetime(2020, 8, 15, 11, 48, 49, 528295)}]
(Background on this error at: http://sqlalche.me/e/2j85)
2020-08-15 11:52:27,250 ERROR   metis                     (pymysql.err.InternalError) (1054, "Unknown column 'transfer_from' in 'field list'")
[SQL: INSERT INTO metis_transfer_info (transfer_from, transfer_to, transfer_hash, status, amount, ctime, utime) VALUES (%(transfer_from)s, %(transfer_to)s, %(transfer_hash)s, %(status)s, %(amount)s, %(ctime)s, %(utime)s)]
[parameters: {'transfer_from': '0x610Bb1573d1046FCb8A70Bbbd395754cD57C2b60', 'transfer_to': '0x1dF62f291b2E969fB0849d99D9Ce41e2F137006e', 'transfer_hash': '0xd43804ab459c9333862158f6a622ed29786b1edf4ffe950a3858d2be6a84ea76', 'status': 1, 'amount': 10000000000000000000, 'ctime': datetime.datetime(2020, 8, 15, 11, 52, 27, 237021), 'utime': datetime.datetime(2020, 8, 15, 11, 52, 27, 237039)}]
(Background on this error at: http://sqlalche.me/e/2j85)
2020-08-15 11:56:07,910 ERROR   metis                     (pymysql.err.DataError) (1264, "Out of range value for column 'amount' at row 1")
[SQL: INSERT INTO metis_transfer_info (transfer_from, transfer_to, transfer_hash, status, amount, ctime, utime) VALUES (%(transfer_from)s, %(transfer_to)s, %(transfer_hash)s, %(status)s, %(amount)s, %(ctime)s, %(utime)s)]
[parameters: {'transfer_from': '0x610Bb1573d1046FCb8A70Bbbd395754cD57C2b60', 'transfer_to': '0x1dF62f291b2E969fB0849d99D9Ce41e2F137006e', 'transfer_hash': '0xb6b3d23f9d2ea3c776957fb79c2da8424192711aba1be9760c266d948f117f82', 'status': 1, 'amount': 10000000000000000000, 'ctime': datetime.datetime(2020, 8, 15, 11, 56, 7, 906444), 'utime': datetime.datetime(2020, 8, 15, 11, 56, 7, 906461)}]
(Background on this error at: http://sqlalche.me/e/9h9h)
2020-08-17 10:57:21,266 ERROR   metis                     ['账户不存在']
2020-08-17 16:20:34,269 ERROR   metis                     ['余额不足']
2020-08-17 21:58:24,299 ERROR   metis                     ['账户不存在']
2020-08-18 10:01:28,171 ERROR   metis                     Eth Error
2020-08-18 10:02:49,556 ERROR   metis                     Eth Error
2020-08-18 10:06:24,719 ERROR   metis                     从区块链获取任务异常
2020-08-18 10:06:24,719 ERROR   metis                     Eth Error
2020-08-18 10:10:50,748 ERROR   metis                     ['账号或密码错误']
2020-08-18 15:52:20,915 ERROR   metis.Home.view           Could not persistence dac
2020-08-18 15:52:24,352 ERROR   metis.Home.view           Could not persistence dac
2020-08-18 15:52:51,283 ERROR   metis.Home.view           Could not persistence dac
2020-08-18 15:52:51,744 ERROR   metis.Home.view           Could not persistence dac
2020-08-18 15:52:51,914 ERROR   metis.Home.view           Could not persistence dac
2020-08-18 15:52:52,102 ERROR   metis.Home.view           Could not persistence dac
2020-08-18 15:52:52,285 ERROR   metis.Home.view           Could not persistence dac
2020-08-18 15:52:52,481 ERROR   metis.Home.view           Could not persistence dac
2020-08-18 15:52:52,648 ERROR   metis.Home.view           Could not persistence dac
2020-08-18 15:53:45,084 ERROR   metis.Home.view           Could not persistence dac
2020-08-18 15:57:04,750 ERROR   metis.Home.view           Could not persistence dac
2020-08-18 19:18:16,277 ERROR   metis                     ['邮箱地址已被注册']
2020-08-20 11:34:10,743 ERROR   metis                     ['该工作无效']
2020-08-20 11:34:34,961 ERROR   metis                     ['该工作无效']
2020-08-20 11:35:08,426 ERROR   metis                     ['该工作无效']
2020-08-20 11:38:39,406 ERROR   metis                     ['该工作无效']
2020-08-20 11:39:58,396 ERROR   metis                     ['该工作无效']
2020-08-20 11:40:43,919 ERROR   metis                     ['该工作无效']
2020-08-24 00:22:17,806 ERROR   metis.Home.view           Could not persistence to db
2020-08-24 00:22:19,777 ERROR   metis.Home.view           Could not persistence to db
2020-08-24 00:22:28,485 ERROR   metis.Home.view           Could not persistence to db
2020-08-24 00:22:52,715 ERROR   metis.Home.view           Could not persistence to db
2020-08-24 00:25:04,234 ERROR   metis.Home.view           Could not persistence to db
2020-08-24 00:25:06,004 ERROR   metis.Home.view           Could not persistence to db
2020-08-24 00:25:37,356 ERROR   metis.Home.view           Could not persistence to db
2020-08-24 00:27:52,607 ERROR   metis.Home.view           Could not persistence to db
2020-08-24 00:28:10,787 ERROR   metis.Home.view           Could not persistence to db
2020-08-24 00:28:18,476 ERROR   metis.Home.view           Could not persistence to db
2020-08-24 00:29:01,461 ERROR   metis                     ['账号或密码错误']
2020-08-24 00:29:07,174 ERROR   metis                     ['账号或密码错误']
2020-08-24 00:29:18,011 ERROR   metis                     ['账号或密码错误']
2020-08-24 08:00:44,307 ERROR   metis.Home.view           Could not persistence to db
2020-08-26 10:58:47,340 ERROR   metis                     ['邮箱地址已被注册']
2020-08-26 12:18:00,145 ERROR   metis                     ['邮箱格式错误']
2020-08-26 12:51:14,005 ERROR   metis                     Eth地址不足
2020-08-26 17:22:39,059 ERROR   metis                     从区块链获取任务异常
Could not identify the intended function with name `takeTask`, positional argument(s) of type `(<class 'str'>, <class 'NoneType'>)` and keyword argument(s) of type `{}`.
Found 1 function(s) with the name `takeTask`: ['takeTask(address,uint256)']
Function invocation failed due to no matching argument types.
2020-08-26 17:22:39,059 ERROR   metis                     Eth Error
2020-08-26 23:05:16,502 ERROR   metis                     ['账户不存在']
2020-08-26 23:06:11,273 ERROR   metis                     ['邮箱地址已被注册']
2020-08-26 23:16:04,214 ERROR   metis                     (pymysql.err.DataError) (1406, "Data too long for column 'work_describe' at row 1")
[SQL: UPDATE metis_work_info SET work_describe=%(work_describe)s, utime=%(utime)s WHERE metis_work_info.id = %(metis_work_info_id)s]
[parameters: {'work_describe': '<span style="font-size: 14.4px;">目的：寻找跑酷爱好者</span><div style="margin: 0px; padding: 0px; font-size: 14.4px;">目标：新增100名成员</div><div style="margin: 0px ... (63 characters truncated) ... le="margin: 0px; padding: 0px; font-size: 14.4px;">时间：2020年9月30日</div><div style="margin: 0px; padding: 0px; font-size: 14.4px;">激励：100 M Token</div>', 'utime': datetime.datetime(2020, 8, 26, 23, 16, 4, 202911), 'metis_work_info_id': 13}]
(Background on this error at: http://sqlalche.me/e/9h9h)
2020-08-26 23:16:10,353 ERROR   metis                     (pymysql.err.DataError) (1406, "Data too long for column 'work_describe' at row 1")
[SQL: UPDATE metis_work_info SET work_describe=%(work_describe)s, utime=%(utime)s WHERE metis_work_info.id = %(metis_work_info_id)s]
[parameters: {'work_describe': '<span style="font-size: 14.4px;">目的：寻找跑酷爱好者</span><div style="margin: 0px; padding: 0px; font-size: 14.4px;">目标：新增100名成员</div><div style="margin: 0px ... (63 characters truncated) ... le="margin: 0px; padding: 0px; font-size: 14.4px;">时间：2020年9月30日</div><div style="margin: 0px; padding: 0px; font-size: 14.4px;">激励：100 M Token</div>', 'utime': datetime.datetime(2020, 8, 26, 23, 16, 10, 348001), 'metis_work_info_id': 13}]
(Background on this error at: http://sqlalche.me/e/9h9h)
2020-08-26 23:16:14,990 ERROR   metis                     (pymysql.err.DataError) (1406, "Data too long for column 'work_describe' at row 1")
[SQL: UPDATE metis_work_info SET work_describe=%(work_describe)s, utime=%(utime)s WHERE metis_work_info.id = %(metis_work_info_id)s]
[parameters: {'work_describe': '<span style="font-size: 14.4px;">目的：寻找跑酷爱好者</span><div style="margin: 0px; padding: 0px; font-size: 14.4px;">目标：新增100名成员</div><div style="margin: 0px ... (63 characters truncated) ... le="margin: 0px; padding: 0px; font-size: 14.4px;">时间：2020年9月30日</div><div style="margin: 0px; padding: 0px; font-size: 14.4px;">激励：100 M Token</div>', 'utime': datetime.datetime(2020, 8, 26, 23, 16, 14, 987947), 'metis_work_info_id': 13}]
(Background on this error at: http://sqlalche.me/e/9h9h)
2020-08-26 23:16:15,418 ERROR   metis                     (pymysql.err.DataError) (1406, "Data too long for column 'work_describe' at row 1")
[SQL: UPDATE metis_work_info SET work_describe=%(work_describe)s, utime=%(utime)s WHERE metis_work_info.id = %(metis_work_info_id)s]
[parameters: {'work_describe': '<span style="font-size: 14.4px;">目的：寻找跑酷爱好者</span><div style="margin: 0px; padding: 0px; font-size: 14.4px;">目标：新增100名成员</div><div style="margin: 0px ... (63 characters truncated) ... le="margin: 0px; padding: 0px; font-size: 14.4px;">时间：2020年9月30日</div><div style="margin: 0px; padding: 0px; font-size: 14.4px;">激励：100 M Token</div>', 'utime': datetime.datetime(2020, 8, 26, 23, 16, 15, 414442), 'metis_work_info_id': 13}]
(Background on this error at: http://sqlalche.me/e/9h9h)
2020-08-26 23:16:39,504 ERROR   metis                     (pymysql.err.DataError) (1406, "Data too long for column 'work_describe' at row 1")
[SQL: UPDATE metis_work_info SET work_describe=%(work_describe)s, utime=%(utime)s WHERE metis_work_info.id = %(metis_work_info_id)s]
[parameters: {'work_describe': '<span style="font-size: 14.4px;">目的：寻找跑酷爱好者</span><div style="margin: 0px; padding: 0px; font-size: 14.4px;">目标：新增100名成员</div><div style="margin: 0px ... (63 characters truncated) ... le="margin: 0px; padding: 0px; font-size: 14.4px;">时间：2020年9月30日</div><div style="margin: 0px; padding: 0px; font-size: 14.4px;">激励：100 M Token</div>', 'utime': datetime.datetime(2020, 8, 26, 23, 16, 39, 502931), 'metis_work_info_id': 13}]
(Background on this error at: http://sqlalche.me/e/9h9h)
2020-08-26 23:17:52,763 ERROR   metis                     (pymysql.err.DataError) (1406, "Data too long for column 'work_describe' at row 1")
[SQL: UPDATE metis_work_info SET work_describe=%(work_describe)s, utime=%(utime)s WHERE metis_work_info.id = %(metis_work_info_id)s]
[parameters: {'work_describe': '<span style="font-size: 14.4px;">目的：寻找跑酷爱好者</span><div style="margin: 0px; padding: 0px; font-size: 14.4px;">目标：新增100名成员</div><div style="margin: 0px ... (63 characters truncated) ... le="margin: 0px; padding: 0px; font-size: 14.4px;">时间：2020年9月30日</div><div style="margin: 0px; padding: 0px; font-size: 14.4px;">激励：100 M Token</div>', 'utime': datetime.datetime(2020, 8, 26, 23, 17, 52, 757772), 'metis_work_info_id': 13}]
(Background on this error at: http://sqlalche.me/e/9h9h)
2020-08-26 23:19:25,110 ERROR   metis                     (pymysql.err.DataError) (1406, "Data too long for column 'work_describe' at row 1")
[SQL: UPDATE metis_work_info SET work_describe=%(work_describe)s, utime=%(utime)s WHERE metis_work_info.id = %(metis_work_info_id)s]
[parameters: {'work_describe': '<span style="font-size: 14.4px;">目的：寻找跑酷爱好者</span><div style="margin: 0px; padding: 0px; font-size: 14.4px;">目标：新增100名成员</div><div style="margin: 0px ... (63 characters truncated) ... le="margin: 0px; padding: 0px; font-size: 14.4px;">时间：2020年9月30日</div><div style="margin: 0px; padding: 0px; font-size: 14.4px;">激励：100 M Token</div>', 'utime': datetime.datetime(2020, 8, 26, 23, 19, 25, 104870), 'metis_work_info_id': 13}]
(Background on this error at: http://sqlalche.me/e/9h9h)
2020-08-26 23:19:48,760 ERROR   metis                     (pymysql.err.DataError) (1406, "Data too long for column 'work_describe' at row 1")
[SQL: UPDATE metis_work_info SET work_describe=%(work_describe)s, utime=%(utime)s WHERE metis_work_info.id = %(metis_work_info_id)s]
[parameters: {'work_describe': '<span style="font-size: 14.4px;">目的：寻找跑酷爱好者</span><div style="font-size: 14.4px; margin: 0px; padding: 0px;">目标：新增100名成员</div><div style="font-size:  ... (63 characters truncated) ... le="font-size: 14.4px; margin: 0px; padding: 0px;">时间：2020年9月30日</div><div style="font-size: 14.4px; margin: 0px; padding: 0px;">激励：100 M Token</div>', 'utime': datetime.datetime(2020, 8, 26, 23, 19, 48, 759010), 'metis_work_info_id': 13}]
(Background on this error at: http://sqlalche.me/e/9h9h)
2020-08-26 23:21:42,003 ERROR   metis                     ['该工作名已存在']
2020-08-31 02:53:53,204 ERROR   metis                     ['账户不存在']
2020-08-31 02:53:58,959 ERROR   metis                     ['账户不存在']
2020-08-31 02:54:03,135 ERROR   metis                     ['邮箱格式错误', '账户不存在']
2020-08-31 02:54:26,588 ERROR   metis                     ['邮箱地址已被注册']
2020-08-31 02:55:26,111 ERROR   metis                     ['邮箱地址已被注册']
2020-08-31 02:55:33,195 ERROR   metis                     ['邮箱地址已被注册']
2020-08-31 02:56:30,183 ERROR   metis                     ['邮箱地址已被注册']
2020-08-31 02:56:35,623 ERROR   metis                     ['邮箱地址已被注册']
2020-08-31 02:56:36,511 ERROR   metis                     ['邮箱地址已被注册']
2020-08-31 02:56:37,361 ERROR   metis                     ['邮箱地址已被注册']
2020-08-31 03:16:49,472 ERROR   metis                     从区块链获取任务异常
Could not identify the intended function with name `takeTask`, positional argument(s) of type `(<class 'str'>, <class 'NoneType'>)` and keyword argument(s) of type `{}`.
Found 1 function(s) with the name `takeTask`: ['takeTask(address,uint256)']
Function invocation failed due to no matching argument types.
2020-08-31 03:16:49,473 ERROR   metis                     Eth Error
2020-08-31 03:17:01,410 ERROR   metis                     从区块链获取任务异常
Could not identify the intended function with name `takeTask`, positional argument(s) of type `(<class 'str'>, <class 'NoneType'>)` and keyword argument(s) of type `{}`.
Found 1 function(s) with the name `takeTask`: ['takeTask(address,uint256)']
Function invocation failed due to no matching argument types.
2020-08-31 03:17:01,413 ERROR   metis                     Eth Error
2020-08-31 03:19:59,260 ERROR   metis                     从区块链获取任务异常
Could not identify the intended function with name `takeTask`, positional argument(s) of type `(<class 'str'>, <class 'NoneType'>)` and keyword argument(s) of type `{}`.
Found 1 function(s) with the name `takeTask`: ['takeTask(address,uint256)']
Function invocation failed due to no matching argument types.
2020-08-31 03:19:59,282 ERROR   metis                     Eth Error
2020-08-31 03:21:57,513 ERROR   metis                     从区块链获取任务异常
Could not identify the intended function with name `takeTask`, positional argument(s) of type `(<class 'str'>, <class 'NoneType'>)` and keyword argument(s) of type `{}`.
Found 1 function(s) with the name `takeTask`: ['takeTask(address,uint256)']
Function invocation failed due to no matching argument types.
2020-08-31 03:21:57,513 ERROR   metis                     Eth Error
2020-08-31 03:45:54,642 ERROR   metis                     ['账户不存在']
2020-08-31 03:45:58,900 ERROR   metis                     ['账户不存在']
2020-08-31 03:52:34,148 ERROR   metis                     ['账号或密码错误']
2020-08-31 03:52:41,374 ERROR   metis                     ['账号或密码错误']
2020-08-31 21:08:19,144 ERROR   metis                     ['账号或密码错误']
2020-09-08 09:19:51,471 ERROR   metis                     ['账号或密码错误']
2020-09-10 00:37:58,032 ERROR   metis                     ['该工作名已存在']
2020-09-10 00:38:08,806 ERROR   metis                     ['该工作名已存在']
2020-09-10 00:38:18,125 ERROR   metis                     ['该工作名已存在']
2020-09-17 20:26:57,812 ERROR   metis                     提交工作到区块链异常
2020-09-21 11:48:19,388 ERROR   metis.Home.view           Could not persistence to db
2020-09-21 11:52:59,998 ERROR   metis.Home.view           Could not persistence to db
2020-09-21 11:53:00,592 ERROR   metis.Home.view           Could not persistence to db
2020-09-21 11:56:27,387 ERROR   metis                     从区块链获取任务异常
Could not identify the intended function with name `takeTask`, positional argument(s) of type `(<class 'str'>, <class 'NoneType'>)` and keyword argument(s) of type `{}`.
Found 1 function(s) with the name `takeTask`: ['takeTask(address,uint256)']
Function invocation failed due to no matching argument types.
2020-09-21 11:56:27,391 ERROR   metis                     Eth Error
2020-09-21 12:25:39,925 ERROR   metis                     从区块链获取任务异常{'code': -32000, 'message': 'replacement transaction underpriced'}
2020-09-21 12:25:39,934 ERROR   metis                     Eth Error
2020-09-21 14:21:57,378 ERROR   metis                     ['账号或密码错误']
2020-09-21 14:23:04,089 ERROR   metis.Home.view           Could not persistence to db
2020-09-21 14:23:23,465 ERROR   metis.Home.view           Could not persistence to db
2020-09-21 14:36:00,027 ERROR   metis                     提交工作到区块链异常
2020-09-21 14:36:06,148 ERROR   metis                     提交工作到区块链异常
2020-09-21 14:37:18,371 ERROR   metis                     ['账号或密码错误']
2020-09-25 21:23:56,751 ERROR   metis.Home.view           Could not persistence to db
2020-09-25 21:23:57,478 ERROR   metis.Home.view           Could not persistence to db
2020-09-25 21:45:40,276 ERROR   metis                     Eth地址不足
2020-09-25 21:45:54,314 ERROR   metis                     Eth地址不足
2020-09-25 22:02:28,746 ERROR   metis                     ['账户不存在']
2020-09-25 22:17:07,455 ERROR   metis                     ['账户不存在']
2020-09-28 10:35:33,820 ERROR   metis                     从区块链获取任务异常
Could not identify the intended function with name `takeTask`, positional argument(s) of type `(<class 'str'>, <class 'NoneType'>)` and keyword argument(s) of type `{}`.
Found 1 function(s) with the name `takeTask`: ['takeTask(address,uint256)']
Function invocation failed due to no matching argument types.
2020-09-28 10:35:33,830 ERROR   metis                     Eth Error
